Library/Homebrew/language/python.rb

```diff
@@ -88,6 +88,17 @@ def self.setup_install_args(prefix)
       ]
     end
 
+    def self.rewrite_python_shebang(python_path)
+      Pathname(".").find do |f|
+        regex = %r{^#! ?/usr/bin/(env )?python([23](\.\d{1,2})?)$}
+        maximum_regex_length = "#! /usr/bin/env pythonx.yyy$".length
+        next unless f.file?
+        next unless regex.match?(f.read(maximum_regex_length))
+
+        Utils::Inreplace.inreplace f.to_s, regex, "#! #{python_path}"
```

Comment:
```suggestion
        Utils::Inreplace.inreplace f.to_s, regex, "#!#{python_path}"
```
Bike shedding, but I'm not personally a fan of the space after the shebang line. It's atypical.